File: annfengshui/remove_pricing.js
const fs = require('fs');

let content = fs.readFileSync('app/data/products.ts', 'utf8');

// Remove pricing sections from fullDescription
content = content.replace(/________________________________________\n💰 GIÁ BÁN \/ PRICING[\s\S]*?________________________________________/g, '________________________________________');

// Remove pricing sections from fullDescriptionEn
content = content.replace(/________________________________________\n💰 PRICING[\s\S]*?________________________________________/g, '________________________________________');

fs.writeFileSync('app/data/products.ts', content);
console.log('Pricing sections removed successfully');

File: annfengshui/remove_pricing.py
import re

# Read the file
with open('app/data/products.ts', 'r', encoding='utf-8') as f:
    content = f.read()

# Remove pricing sections from fullDescription
content = re.sub(r'________________________________________\n💰 GIÁ BÁN / PRICING[\s\S]*?________________________________________', '________________________________________', content)

# Remove pricing sections from fullDescriptionEn
content = re.sub(r'________________________________________\n💰 PRICING[\s\S]*?________________________________________', '________________________________________', content)

# Write back
with open('app/data/products.ts', 'w', encoding='utf-8') as f:
    f.write(content)

print("Pricing sections removed successfully")